public/js/monitor.js

```diff
@@ -1,4 +1,56 @@
 "use strict";
+/* global ga */
+
+function sendEvent(category, action, label){
+  if(ga){
+  ga("send", "event", category, action, label);
+  }
+}
+
+function handleSignUpClicks(string){
+  const eventCategory = "Sign Ups";
+  const eventAction = "Clicked Sign Up";
+  if(window.location.href.indexOf("breach") > -1){
+    sendEvent(eventCategory, eventAction, "Sign up button clicked from featured breach.");
+  }
+  else if (document.getElementById("what-to-do")) {
+    sendEvent(eventCategory, eventAction, "Sign up button clicked after scanning email with breaches.");
+  }
+  else {
+    if(window.location.href.indexOf("scan") > -1){
```

Comment:
Same here. I think this scares me.

If it works, it's by chance and magic.
Consider https://fx-breach-alerts.herokuapp.com/?breach=Adobe
```js
console.log(window.location.href.indexOf("scan")); // -1
```

But if a new breach for the imaginary "Adobescan" site appears tomorrow, we may get some unexpected results (or not, I'm just being delightfully hypothetical):

https://fx-breach-alerts.herokuapp.com/?breach=Adobescan

```js
console.log(window.location.href.indexOf("scan")); // 52
```

It'd also break if we ever added some "/scanner" URL or something else vague.

I think this is fine today, because I didn't see any breaches with "scan" in the name in the current breaches.json:

```sh
$ cat ../blurts-addon/src/breaches.json | grep scan # returns zero results
```

Reply:
You're right. Going to put IDs on the resulting h2s and check for those instead. Still cheesy, but feels less perilous for the meantime. Less magic, more muggle.